tests/test_chromatic.py

```diff
@@ -1437,7 +1437,10 @@ def __repr__(self):
         parallactic_angle=0.*galsim.degrees)
     chrom_shear = lambda w: galsim.Shear(g1=0.2+0.2*(w-500.)/500.,g2=0.) if w<1000. else \
         galsim.Shear(g1=0.4, g2=0.)
-    chrom_shift_y = lambda w: scale*(w-500.)
+    # The 0.5 in the line below used to be 1.0.  When we improved the .thin() method, this test
+    # started to fail in 0.1% of pixels.  Making the chromaticity less extreme allowed the test
+    # to pass.
+    chrom_shift_y = lambda w: 0.5*scale*(w-500.)
```

Comment:
Also, I'm a little concerned about this change.  Especially since the change to the thin() method should have been an improvement in accuracy.  So do you know why this started failing?  Was this just a case where the old thinning algorithm was actually more accurate for this particular case?  If so, should we maybe use a lower rel_err on the thin() call, rather than change the magnitude of the chromatic shift here?